src/modules/lodepng.cpp

```diff
@@ -0,0 +1,6961 @@
+#include "lodepng.h"
```

Comment:
Put this file somehwere else. Maybe in a `thrid_party` directory to avoid mixing it with ours.